bitnami/fluent-bit/templates/daemonset.yaml

```diff
@@ -0,0 +1,175 @@
+{{- if .Values.useDaemonset }}
```

Comment:
This value is not declared in the values file. Did you mean `daemonset.enabled`?

Reply:
Fixed [5820749](https://github.com/bitnami/charts/pull/15453/commits/5820749206c330a85f05bf0c5b6ebce39ab0a235)